test/extended/deployments/deployments.go

```diff
@@ -545,6 +545,13 @@ var _ = g.Describe("[sig-apps][Feature:DeploymentConfig] deploymentconfigs", fun
 
 		g.It("should run a successful deployment with multiple triggers", func() {
 			g.By("creating DC")
+
+			_, err := oc.Run("import-image").Args("registry.redhat.io/ubi8/ruby-30:latest", "--confirm", "--reference-policy=local").Output()
```

Comment:
Shouldn't we use only images that are mirrored? https://github.com/openshift/origin/blob/master/test/extended/util/image/README.md

Reply:
i'm trying to determine the answer to that here:

https://github.com/openshift/origin/pull/26912

because many of these tests already weren't using images that were mirrored (this one, for example, was just using the ruby imagestream provided by the samples operator and hosted on registry.redhat.io already).

so this was the minimal change to get away from using that imagestream while still using the same underlying image.

since i'm not confident we should be mirroring images like this (redhat product images), i'm hoping we are allowed use direct references for non-conformance tests.

Reply:
@dmage https://github.com/openshift/origin/pull/26912 has clarified that only conformance tests need their images mirrored, these are not conformance tests.